std/net/curl.d

```diff
@@ -1,6 +1,10 @@
 // Written in the D programming language.
 
 /**
+$(MESSAGE_BOX gray, $(LINK2 https://github.com/ikod/dlang-requests, `requests`) provides
+a simple, high-level API with a native D implementation. Moreover, `requests`
+can be used with $(LINK2 http://vibed.org/, Vibe.d) for asynchronous requests.)
```

Comment:
I actually would like to go with `red` here as it's like `std.xml` on the hit list and even has an excellent, well-maintained replacement!